modules/Emails/mail.php

```diff
@@ -467,6 +467,23 @@ function addAttachment($mail, $filename, $record) {
 	}
 }
 
+/** Function to check if a number is a attachmentid
+  * $id -- number or attachmentid
+  */
+function is_attachmentid($id) {
```

Comment:
Should this function be in utils.php or CommonUtils.php in case we need to use it somewhere else? In fact, did you check if there is already a function that may do this?

Reply:
I had a look, couldn't find any function with this functionality. The closest one is [isRecordExists]8https://github.com/tsolucio/corebos/blob/master/include/utils/utils.php#L3312). I would suggest we move this function after (or before) that one.